GRDB/QueryInterface/Schema/TableDefinition.swift

```diff
@@ -619,6 +619,19 @@ public final class ColumnDefinition {
         var deferred: Bool
     }
     
+    /// The GeneratedColumnQualification enum defines whether a generated column is virtual or stored
```

Comment:
Just realized this line should end with a period. 🤦

Reply:
😅